validator/java/BUILD

```diff
@@ -0,0 +1,74 @@
+sh_binary(
+    name = "copyValidatorJavaSource",
+    srcs = ["src/main/bin/copyValidatorJavaSource.sh"],
+    deps = [
+        "@bazel_tools//tools/bash/runfiles"
+    ],
+)
+
+sh_binary(
+    name = "fetchAMPResources",
+    srcs = ["src/main/bin/fetchAMPResources.sh"],
+    deps = [
+        "@bazel_tools//tools/bash/runfiles"
+    ],
+)
+
+load("@rules_java//java:defs.bzl", "java_proto_library")
+load("@rules_proto//proto:defs.bzl", "proto_library")
+
+java_proto_library(
+    name = "amphtml_validator_java_proto_lib",
+    visibility = ["//visibility:public"],
+    deps = [":amphtml_validator_proto_lib"],
+)
+
+proto_library(
+    name = "amphtml_validator_proto_lib",
+    srcs = ["src/main/proto/validator.proto"],
```

Comment:
Can you add the following in src/main/proto/validator.proto so that the java package is consistent with other files.

option java_package = "dev.amp.validator";
option java_outer_classname = "ValidatorProtos";

and then update all the imports of validator proto in java files.

Thanks!

Reply:
@amaltas I can do that, I am unsure if you're implying validator.proto should be included in the PR though. How will this change be reflected if we currently git ignore the file?

Reply:
Is validator.proto in .gitignore file? You can remove it. I think it is ok to add the validator.proto in this PR.

Once you add these options field in proto java_proto_library will generate proto classes in dev.amp.validator package, requiring you to update the import statements.

Reply:
yes, okay, we will do that, with the understanding that the changes to the validator.proto within the amphtml/validator directory is not guaranteed to match the file in our subdirectory.